### PR TITLE
[CI] run operator tests with naive engine

### PR DESCRIPTION
**Pull request by szha**
## Description ##
run operator tests with naive engine

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] To the best of my knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] run operator tests with naive engine

### CODE REVIEW

File: tests/python/unittest/test_numpy_op.py [hunk 1]
```diff
@@ -1326,7 +1325,7 @@ def hybrid_forward(self, F, a):
 
 @with_seed()
 @use_np
-@unittest.skip("NumpyBooleanAssignForwardCPU broken: https://github.com/apache/incubator-mxnet/issues/17990")
+@pytest.mark.skip(reason="NumpyBooleanAssignForwardCPU broken: https://github.com/apache/incubator-mxnet/issues/17990")
```
Comment (leezu) [inline tests/python/unittest/test_numpy_op.py:1328]: This should haven been fixed by https://github.com/apache/incubator-mxnet/pull/18158 so let's remove the skip

**Comment by marcoabreu** [comment]
What's our strategy to test multi threaded engine?

**Comment by szha** [comment]
@marcoabreu all but the operator unit tests still run on threaded engine.

**Comment by mseth10** [comment]
hey @szha , this commit breaks the nightly CD pipeline.

tests/python/unittest/test_extensions.py::test_subgraph should be SKIPPED, but after this commit it is not getting skipped.

Basically, this change is not working as expected: https://github.com/apache/incubator-mxnet/commit/10b6b4887ef494ea985c9ea75a393a985476d08e#diff-460d06cb3af2af996c5ec259b086712fR100

Can you please fix this?

**Comment by DickJC123** [comment]
I realize this PR is over a year old, but does anyone remember the original problem being solved, i.e. why can't the operator unittests run with the threaded_engine_per_device?  Seems like if we're only running the unittests once, it should be with the default engine.

**Pull request merged**